src/help/fileContainer.ts

```diff
@@ -13,14 +13,24 @@ export const uploadContentToFileContainer = async (
   batch: BatchUpdateDocuments
 ) => {
   const uploadURL = new URL(fileContainer.uploadUri);
-  return axios.put(
-    uploadURL.toString(),
-    {
-      addOrUpdate: batch.addOrUpdate.map((docBuilder) => docBuilder.marshal()),
-      delete: batch.delete,
-    },
-    getFileContainerAxiosConfig(fileContainer)
-  );
+  return axios
+    .put(
+      uploadURL.toString(),
+      {
+        addOrUpdate: batch.addOrUpdate.map((docBuilder) =>
+          docBuilder.marshal()
+        ),
+        delete: batch.delete,
+      },
+      getFileContainerAxiosConfig(fileContainer)
+    )
+    .catch((err) => {
+      if (isMaxBodyLengthExceededError(err)) {
+        err.message +=
+          '\nTry setting and/or increasing `axios.defaults.maxBodyLength`.';
```

Comment:
Can't we increase the axios default max body length to maximum supported by a file container?

Reply:
If there's a limit that is enforced, yeah sure. My goal here was to not draw any line in the sand, because I was not sure of what the actual technical limits are.